components/BlogPost.module.css

```diff
@@ -0,0 +1,20 @@
+.blog-post {
+}
+
+.blog-post__media {
+}
+
+.blog-post__content {
+}
```

Comment:
If you don't need it, you should remove it. 😛

Reply:
Got it! ;)